### PR TITLE
Show detail orders in admin page

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/43421)

## WHAT (optional)
- Show detail orders in admin page

## HOW
- Add new def **show**, **load_order** in orders controller
- Add new view admin/orders/show.html.erb and _order_details.html.erb
- Change i18n in file locales

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: app/helpers/admin/orders_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module Admin::OrdersHelper
```
Comment (quoctd-0607) [inline app/helpers/admin/orders_helper.rb]: không dùng thì bỏ file này đi e

**Comment by ghost** [comment]
ready

File: app/helpers/admin/orders_helper.rb (hunk 1 above)
Comment (ghost) [reply app/helpers/admin/orders_helper.rb]: dạ anh

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**